bundles/org.openhab.binding.argoclima/scripts/download_ota_fw_files.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+"""
+Downloads current Argo Ulisse firmware binary files from manufacturer's servers
+"""
+
+__author__ = "Mateusz Bronk"
+__copyright__ = """Copyright (c) 2010-2024 Contributors to the openHAB project
```

Comment:
This could cause some problems, see #16168. It will not be automatically updated. Is it needed?

Reply:
Happy to conform to any overarching rules the project has. From my POV this may not be harmful though. Some thoughts below (esp see pt. 3).
Personally - I'd leave as-is. If you believe it must be changed, I can follow in any firm direction you indicate. Just let me know.

1. **Why have this script here in the first place?** 
    - It's a developer aid _(to get the latest FW to play with and possibly extend this binding to also allow FW updates)_. Also helps a savvy user to see what's the latest. Having it in this repo is not a must, but I don't have a good place for it (too small to warrant starting a separate GH project) and not including at all... is harmless... but a step back IMHO (it has some value)

1. **Does it need to have CP headers?**
    - No. I'm just so used to the "_every 'code' file needs a copyright stanza_" I added it w/o thinking twice. Of course, couldn't add the `java` format, so went with _pythonic_ way of expressing it. If you say it can be removed, I have no issue removing the CP entirely

1. **Is an out-of-date year in the copyright header an issue here?**
    - I am not a lawyer - but from my practice: **no** (unless the file changes significantly). Not an authoritative source, but see this as well: https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated 
    Overall, yes, in my projects I also do a "fire and forget (update all CP year)", b/c it is by far the easiest. But for this particular file... it's just an useful script/snippet - I don't expect it getting much maintenance in the future.

Reply:
The issue is that we use `mvn license:format` to update all copyright headers project-wise when entering a new year. We also have SAT for detecting wrong copyright years. These tools are not configured to include `.py` files or partial headers like this one. So it will effectively be left unmaintained. I don't think we need it for this supporting script, so if you are fine with removing it, let's do this to avoid doubt and/or manual work when checking the result of the automated header updates.

Reply:
I hear you. Outliers and people wasting  time, wondering if it can be left w/o update is a bad thing. One of the other options would be a copyright without a year (I've seen and used this format also in professional settings, but can't say if it is any 'legal').

Just have a small favor to ask - since I won't have access to my PC during the weekend, would you mind adding this edit (CP removal) as a GH suggestion, so I can commit it from my phone? Can also wait till Monday of course, but if there's any feature/code freeze approaching, maybe better to get ahead of the curve?

Reply:
This copyright can be corrected after feature freeze.

Reply:
Removed - please check if this is what you had in mind.